server/src/services/heatmapService.py

```diff
@@ -28,7 +28,7 @@ async def get_heatmap(self, filters):
                 filters['requestTypes'],
                 filters['ncList'])
 
-            pins = dataAccess.query(fields, filters)
+            pins = dataAccess.query(fields, filters, table='map')
```

Comment:
We should probably put in a table existence safety check somewhere in the dataAcess layer, I'm cool with it going in after this PR but it should prob go in at some point

Reply:
yeah good call, I'll make a ticket